playbooks/host/hosts-present-with-managedby_host.yml

```diff
@@ -4,7 +4,7 @@
   become: true
 
   tasks:
-    ipahost:
+    - ipahost:
```

Comment:
The indentation here differs from the playbooks above.

Reply:
Fixed.